sgl-kernel/setup.py

```diff
@@ -24,7 +25,8 @@ def update_wheel_platform_tag():
         old_wheel.rename(new_wheel)
 
 
-cutlass = root / "3rdparty" / "cutlass"
+cutlass_default = root / "3rdparty" / "cutlass"
+cutlass = Path(os.environ.get("VLLM_CUTLASS_SRC_DIR", default=cutlass_default))
```

Comment:
It seems more reasonable to name it `CUSTOM_CUTLASS_SRC_DIR` .

Reply:
yes, let us use `CUSTOM_CUTLASS_SRC_DIR`